### ISSUE TITLE
Subject values: evaluate relevancy of the existing list or update

### ISSUE THREAD

**Issue by zzacharo** [CERNDocumentServer/cds-rdm#249]
- In the submission form, there is [this list](https://cds.cern.ch/kb?kb=54) (you should have access?) of subjects values. 
- Is this up-to-date? Should we review the available values? 
- Any relevant source at CERN for this?
- Provided from @michamos the list that exists in INSPIRE(https://mattermost.web.cern.ch/it-dep/pl/cqi8hjzddf8kjkoxxro4r7bq6o)

**Comment by PaulinaBaranowska** [comment]
I will also look into this

**Comment by PaulinaBaranowska** [comment]
- [X] Add missing subjects to CDS KB (Quantum Technology & Education and Outreach)
- [ ] Find records with subjects in CDS that have values outside of this list (for CDS to provide to SIS)
- [ ] Clean the values (SIS)

**Comment by PaulinaBaranowska** [comment]
Questions:
- Other => XX (should it be the other way round? We do not need it as we already have `Other Subjects`?)
- Particle Physics-Experiment is twice in CDS KB (once as `AB` and once as `e`), e should be deleted.

**Comment by michamos** [comment]
@ntarocco do you mean it won't be a controlled vocabulary? or it will be but there is also an escape hatch?

**Comment by zzacharo** [comment]
> [@ntarocco](https://github.com/ntarocco) do you mean it won't be a controlled vocabulary? or it will be but there is also an escape hatch?

It means that in the system if a user doesn't find the value in the controlled vocabulary they can always add it as free text.

**Comment by michamos** [comment]
I don't think we want to allow free text subjects here, that defeats the purpose. Can't they use keywords for that? or are subjects and keywords the same thing? It might be useful if you give some more info on how this would look in the schema.

**Comment by zzacharo** [comment]
In the new system, user will autocomplete from the subjects vocabulary and if they do not find what they are looking for then they add it as free text. We store both the controlled values in the `subjects` field as follows:
```JSON
"subjects": [{
    "id": "Accelerators and Storage Rings",
    "subject": "Accelerators and Storage Rings",
    "scheme": "CERN"
  },
  {
    "subject": "myvalue"
}],
```
Subjects without a specific `id` are considered keywords. These are shown at the moment like below in the record's detail page:

![Screenshot 2024-11-14 at 09 57 33](https://github.com/user-attachments/assets/ddcc1a80-674a-4059-b418-012489546903)

You can see for example this record: https://dev-cds-rdm.web.cern.ch/records/mddtr-zvt57

**Comment by michamos** [comment]
Thanks @zzacharo for the explanation. Note that we rely on the subjects for the annual report stats (see bottom diagram on [the dashboard](https://bi.siscern.org/superset/dashboard/annual-reports/?native_filters_key=_L-WUePg0ntdu0AFvSsGeFW4HwBAYxBFK6ASJLQT3jLjH4DO0LDp1oKOFMffR4wP), but not sure to what extent we care about those for the rest of CDS. We will discuss further with @agentilb.

**Comment by kpsherva** [comment]
this subject will need a follow up in the other collections - based on their metadata. For now on hold
